docs/UserGuide.md

```diff
@@ -178,8 +183,10 @@ Examples:
 
 ### Adding a label to a tag : `label`
 
-Adds one or more label to an existing tag.<br>
-You can use label to categorise tags.
+Adds one or more labels to an existing tag.<br>
+You can use label to categorise tags so that you can open all the tags with the same label simultaneously.
```

Comment:
Currently I feel that the purpose of labelling is to open multiple files, based on the description. But I think the main purpose of label should still be categorising tags, and open label is an additional feature which should not be emphased here.

Reply:
Changed. I put categorizing tags as the main purpose and opening multiple files as a side feature.